src/index.html

```diff
@@ -2,11 +2,398 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com/">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="images\logo.png" alt="Logo Moyo">
+      </a>
+
+      <nav class="header__navigation">
+        <ul class="navigation">
+          <li class="navigation__link">
+            <a href="#apple" class="navigation__item is-active">Apple</a>
+          </li>
+
+          <li class="navigation__link">
+            <a href="#samsung" class="navigation__item">Samsung</a>
+          </li>
+
+          <li class="navigation__link">
+            <a href="#smartphones" class="navigation__item">Smartphones</a>
+          </li>
+
+          <li class="navigation__link">
+            <a
+              href="#laptops&computers"
+              class="navigation__item"
+              data-qa="nav-hover"
+              >Laptops & Computers</a>
```

Comment:
DOn't forget about code style, fix it in all cases
```suggestion
            <a
              href="#laptops&computers"
              class="navigation__item"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
```